commcare-cloud/commcare_cloud/commcare_cloud.py

```diff
@@ -306,6 +325,7 @@ def make_parser(parser):
         parser.add_argument('--become-user', help=(
             "run operations as this user (default=root)"
         ))
+        arg_skip_check(parser)
```

Comment:
Does this mean `run-shell-command` now defaults to running in check mode first? If so, I'd suggest modifying it to always pass in `--skip-check`, since the shell module's check mode is useless as far as I know

Reply:
good point